packages/test/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from 'enzyme-to-json';
```

Comment:
Won't you prefer to export it using a specific name?
You can achieve that by `export { default as enzymeToJson } from '...';`